src/dissect/dissect.c

```diff
@@ -601,6 +601,8 @@ static int action_dissect(DissectedImage *m, LoopDevice *d) {
         else if (arg_json_format_flags & JSON_FORMAT_OFF)
                 printf("      Size: %s\n", FORMAT_BYTES(size));
 
+        printf(" Sec. Size: %" PRIu32 "\n", m->sector_size);
```

Comment:
Why abbreviate here? I'd just write sector size in full here

Reply:
then the `:` wouldn't be aligned to the `:` of the previous and next lines!